rpc/src/main/scala/MasterClient.scala

```diff
@@ -18,6 +18,17 @@ object MasterClient {
     val blockingStub = MasterGrpc.blockingStub(channel)
     new MasterClient(channel, blockingStub)
   }
+
+  def main(args: Array[String]): Unit = {
+    val client = MasterClient("localhost", 50051)
```

Comment:
We should think about how to get master's host and port.

Reply:
MasterClient.scala is working on worker node.  So we get those information from cli.